ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/Utils.java

```diff
@@ -196,6 +196,23 @@ public static boolean shouldReplicate(ReplicationSpec replicationSpec, Table tab
         }
         return shouldReplicateExternalTables;
       }
+
+      if (AcidUtils.isTransactionalTable(tableHandle.getTTable())) {
+        // For testing purposes only, do not replicate ACID tables when config says so
+        boolean shouldReplicateAcidTables = true;
+        if (hiveConf.getBoolVar(HiveConf.ConfVars.HIVE_IN_TEST_REPL)) {
```

Comment:
Need not check HIVE_IN_TEST_REPL anywhere. Let's have the proper support of REPL_DUMP_INCLUDE_ACID_TABLES flag.

Reply:
The idea is to use this flag only for testing and not expand its usage beyond testing. So, as per discussion I have removed this flag from HiveConf and defined in ReplUtils. We still use HIVE_IN_TEST_REPL to protect dump in case the user is using REPL_DUMP_INCLUDE_ACID_TABLES in its config.